### ISSUE TITLE
.env file not watched by dev server

### ISSUE THREAD

**Issue by Philzen** [redwoodjs/redwood#5825]
When doing any changes to the `.env` file, rebuild + reload is not being triggered.

Should be an easy fix, because it is definitely part of the live rebuild + reload, b/c as soon as i change any other file (e.g. any line in any component's code), changes in the `.env` file become active.

**Comment by Philzen** [comment]
Ideally, changes in `.env.defaults` would also be reflected, however as i couldn't get them to become applied unless doing a full server restart, so i believe that may be harder to achieve. May be worth looking into, but for the time being i'd be just happy with `.env` to be added to the watch-list.

**Comment by dac09** [comment]
Hi @Philzen thanks for raising this!

I think… but am not 100% sure, when you change dotenv files webpack doesn’t pick up the changes until you restart. 

While we could certainly do it for the API side easily, it would be a confusing experience if the web and api behave differently. We would need to maintain consistency between both sides, and an unsure of the effort in doing it for the web side.

**Comment by joconor** [comment]
There’s a basic problem with the use of dotenv/dotenv-defaults by RedwoodJS: They will not overwrite already set environment variables. I’ve filed two separate issues that have this as their root cause, and I expect this will complicate efforts to get any ‘watch’ of `.env`, `.env.defaults` correct

**Comment by Philzen** [comment]
> I think… but am not 100% sure, when you change dotenv files webpack doesn’t pick up the changes until you restart.

@dac09 yeah you're right, i must have been mistaken when i opened up the issue, which is now updated accordingly. I guess, until this becomes possible i will have to hard-code and wrap my configs during development.

>  I’ve filed two separate issues that have this as their root cause

@joconor in this repository or over at [motdotla/dotenv](https://github.com/motdotla/dotenv)? If it's the former, i will close this issue as a duplicate. In any case, kindly post the issue links here for reference.

**Comment by jtoar** [comment]
Confirming that changes to `.env` don't get hot reloaded for either the web or api side. @Philzen it's ok to leave this one open as it's a feature request.
